https://github.com/HewlettPackard/netperf/pull/46
https://bugs.gentoo.org/706694

From c6a2e17fe35f0e68823451fedfdf5b1dbecddbe3 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 12 Aug 2020 09:57:23 -0700
Subject: [PATCH] nettest_omni: Remove duplicate variable definitions

These defines are already defined in nettest_bsd.c and exported by
nettest_bsd.h this should fix build with -fno-common

Signed-off-by: Khem Raj <raj.khem@gmail.com>
--- a/src/nettest_omni.c
+++ b/src/nettest_omni.c
@@ -458,14 +458,6 @@ static int client_port_max = 65535;
 
  /* different options for the sockets				*/
 
-int
-  loc_nodelay,		/* don't/do use NODELAY	locally		*/
-  rem_nodelay,		/* don't/do use NODELAY remotely	*/
-  loc_sndavoid,		/* avoid send copies locally		*/
-  loc_rcvavoid,		/* avoid recv copies locally		*/
-  rem_sndavoid,		/* avoid send copies remotely		*/
-  rem_rcvavoid; 	/* avoid recv_copies remotely		*/
-
 extern int
   loc_tcpcork,
   rem_tcpcork,

